test/e2e/pods.go

```diff
@@ -617,9 +617,10 @@ var _ = Describe("Pods", func() {
 			Spec: api.PodSpec{
 				Containers: []api.Container{
 					{
-						Name:    "env3cont",
-						Image:   "gcr.io/google_containers/busybox",
-						Command: []string{"sh", "-c", "env"},
+						Name:            "env3cont",
+						Image:           "gcr.io/google_containers/busybox",
```

Comment:
I don't have permission to push to gcr.io/google_containers, that's why I filed #20386 for that.